### PR TITLE
Update linting tools

### PR DESCRIPTION
**Pull request by JureZmrzlikar**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -33,13 +33,13 @@ commands =
     docs: python setup.py build_sphinx --fresh-env --warning-is-error
 
     # Linter commands:
-    linters: pylint resdk
+    linters: pylint src
```
Comment (dblenkus) [inline tox.ini]: You also want to lint tests directory.

File: setup.py [hunk 2]
```diff
@@ -28,9 +28,8 @@
     url=about['__url__'],
     license=about['__license__'],
     # Exclude tests from built/installed package.
-    packages=setuptools.find_packages(
-        exclude=['tests', 'tests.*', '*.tests', '*.tests.*']
-    ),
+    packages=setuptools.find_packages("src", exclude=["tests", "tests.*", "*.tests", "*.tests.*"]),
```
Comment (dblenkus) [inline setup.py:31]: Is the exclude required as you moved tests out of `src`?

Comment (JureZmrzlikar) [reply setup.py:31]: I'm not sure, i've copied this from [DRF-reactive](https://github.com/genialis/django-rest-framework-reactive/blob/master/setup.py) that has the same structure :-)

Comment (JureZmrzlikar) [reply setup.py:31]: Thanks for this comment. If i remove it all works fine, so i'll just leave it without exclude.

**Pull request merged**